docs/sdks/client-sdks/javascript.md

```diff
@@ -70,16 +70,34 @@ If you are using the SDK for experiment assignments, make sure to pass in an ass
+| **`persistentStore`** (IAsyncStore)               | A custom implementation of an asynchronous, persistent storage for caching flag configurations                                                                                                                                                                                                                                         | _Eppo provided_ |
+| **`maxCacheAgeSeconds`** (number)                 | Maximum age, in seconds, previously cached configurations are considered valid until new ones will be fetched                                                                                                                                                                                                                          | `0`             |
+| **`useExpiredCache`** (boolean)                   | Consider initialization successfully complete without fetching updates, even if the configuration loaded from the cache is expired                                                                                                                                                                                                     | `false`         |
+| **`updateOnFetch`** (always\|expired\|empty)      | Sets how the configuration is updated after a successful fetch:<br/>• always - immediately start using the new configuration<br/>• expired - immediately start using the new configuration only if the current one has expired<br/>• empty - only use the new configuration if the current one is both expired and uninitialized/empty | `'always'`      |
+
+#### Configuration cache
+The SDK can cache previously loaded configurations for use in subsequent sessions, speeding up the time for the SDK to initialize.
+By default, a Local Storage-based cache is used in a browser environment and a Chrome Storage-based cache is used in a Chrome extension environment.
+A custom cache can be provided as the `persistentStore` initialization option.
+
+Caches have a concept of being "expired," dictated by the `maxCacheAgeSeconds` initialization option, and also overridable by the implementation of `isExpired()` for a custom persistent store.
+When the cached configuration is not expired, it is considered valid and no fetches will be made for updated configurations.
+
+When fetches are made--either from initialization or regular polling--you can control when the changes take effect with the `updateOnFetch` initialization option.
+It's default of `'always'` will start serving values from the most up-to-date configuration immediately. `'expired'` will only update if the cached configuration--now being used to serve assignments--has expired. 
```

Comment:
Note: practically, since we don't fetch when the cache is not expired, `'always'` and `'expired'` are the same for now. However, if we layer in the ability to fire off one-of fetches or a new option for whether or not to fetch if the cache is expired, then this will matter more.